Tasks/AzureCLIV1/task.json

```diff
@@ -20,7 +20,7 @@
     "version": {
         "Major": 1,
         "Minor": 152,
-        "Patch": 2
+        "Patch": 3
     },
```

Comment:
Update the message in this file for variable; addSpnToEnvironment

Reply:
This is the base message that is used when translating for other languages.

Reply:
Fixed it